Testing/SystemTests/lib/systemtests/systemtesting.py

```diff
@@ -700,6 +744,12 @@ def asString(self, clean=False):
         code += "systest.cleanup()\nsys.exit(exitcode)\n"
         return code
 
+    def asStringSingleThread(self, clean=False):
+        code = self.asString(clean)
```

Comment:
Would it be worth just adding a parameter to `asString` to not add the `sys.exit` rather than having to trim it off later?